docs/rules/class-order.md

```diff
@@ -0,0 +1,21 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/class-order
+description: enforce classnames order
+---
+# vue/class-order
+> enforce classnames order
+
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
```

Comment:
Could you add some sample code?

https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/attributes-order.md

Reply:
done